### ISSUE TITLE
No-relative-imports - allow-siblings flag not working

### ISSUE THREAD

**Issue by robgraeber** [Microsoft/tslint-microsoft-contrib#778]
### Bug Report

- __tslint-microsoft-contrib__: 6.0.0
- __TSLint version__: 5.12.0
- __TypeScript version__: 3.2.2
- __Running TSLint via__: CLI + VSCode

#### TypeScript code being linted

```ts
import { chunkArray } from './Utils';

chunkArray([1, 2, 3], 1);
```

with `tslint.json` configuration:

```json
"no-relative-imports": [true, "allow-siblings"],
```

#### Actual behavior
It's throwing an error for no-relative-imports.
#### Expected behavior
It would respect the "allow-siblings" flag and not be an error.

The weird thing about this is that it seemed to work correctly for me for a day, then the next day it stopped respecting the allow-siblings flag. I verified my node_modules were up to date. This is a very stripped down version of my project that seems to have the issue, run `npm install && npm test` and I get: 
```ERROR: /Users/bobby/Code/test-proj/src/Example.ts:1:1 - Imported module is being loaded from a relative path. Please use an absolute path: import { chunkArray } from './Utils';```

[test-proj.zip](https://github.com/palantir/tslint/files/2737652/test-proj.zip)

**Comment by IllusionMH** [comment]
@robgraeber thank you for raising issue. I will take a look on it.

**Comment by robgraeber** [comment]
@IllusionMH now that I think about it, it could be an issue with tslint-config-airbnb since they're still on 5.2.1.

**Comment by robgraeber** [comment]
I think tslint-config-airbnb was the culprit, apologies for the false alarm.

**Comment by tristan-mastrodicasa** [comment]
**Temporary Fix**
If you want to use the allow-siblings flag right now with  tslint-config-airbnb you can do what I did below

tslint.override.json
`{
  "rulesDirectory": ["../node_modules/tslint-microsoft-contrib"],
  "rules": {
    "no-relative-imports": [true, "allow-siblings"]
  }
}`

tslint.json
`{
  "extends": ["./tslint.override.json", "tslint-config-airbnb"],
  "rules": { ... }
}`

Make sure the override is before the tslint-config-airbnb
